device/dell/x86_64-dellemc_s5248f_c3538-r0/default_sku

```diff
@@ -1 +1 @@
-DellEMC-S5248f-P-25G t1
\ No newline at end of file
+DellEMC-S5248f-P-25G-DPB t1
```

Comment:
@arunlk-dell  with change it will defaults to DPB sku. does it still bootup successfully with out breakout ?

Reply:
yes, it will bootup successfully with default breakouts as per port_config.ini and hwsku.json values.

Reply:
@srideepDell instead of adding new sku, added the DPB support to the existing sku.